integrations/react-ts/src/util.ts

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+
+// Detailed payment intent request and response type can be found here:
+// https://www.airwallex.com/docs/api#/Payment_Acceptance/Payment_Intents/_api_v1_pa_payment_intents_create/post
+type PaymentIntentRequest = {
+  request_id: string;
+  merchant_order_id: string;
+  amount: number;
+  currency: string;
+  order: {
+    products: {
+      url: string;
+      name: string;
+      desc: string;
+      unit_price: number;
+      currency: string;
+      quantity: number;
+    }[];
+  };
+};
+
+type PaymentIntentBody = {
+  amount: number;
+  client_secret: string;
+  id: string;
+  currency: string;
+};
+
+export const createPaymentIntent = async (paymentIntent: PaymentIntentRequest): Promise<PaymentIntentBody> => {
+  const intentData = await axios.post(
```

Comment:
I don't think it's a good idea to write this in a our demo.

Reply:
Maybe use a const to define this url add some comment to let the merchant know that they should use their own endpoint.